aws/main.mk

```diff
@@ -4,27 +4,74 @@ ifndef AWS_REGION
 $(error Please set AWS_REGION via `export AWS_REGION=<aws_region>` or use direnv. This is nessesary for additional tools that are not able to read a region from your AWS profile)
 endif
 
+ifeq ($(AWS_MFA_ENABLED),"true")
+export AWS_ACCESS_KEY_ID=$(shell echo $(MFA_AWS_ACCESS_KEY_VALUE))
+export AWS_SECRET_ACCESS_KEY=$(shell echo $(MFA_AWS_SECRET_ACCESS_KEY_VALUE))
+export AWS_SESSION_TOKEN=$(shell echo $(MFA_AWS_SESSION_TOKEN_VALUE))
+endif
 # Macroses
 ########################################################################################################################
 # We don't check for AWS_PROFILE, but instead we assume the profile name.
 # You can override it, although it's recommended to have a profile per environment in your ~/.aws/credentials
 AWS_PROFILE ?= $(NAMESPACE)-$(ENV)
+AWS_CLI_PROFILE ?= $(shell echo $$(if [ "$(AWS_MFA_ENABLED)" = "true" ]; then echo ""; else echo "--profile $(AWS_PROFILE)"; fi))
```

Comment:
Do you think this should be reflecting the `MFA` nature? like `AWS_PROFILE_MFA`?

Reply:
@AutomationD I do not think so since if we use MFA - there is no needs in AWS Profile, thoughts?

Reply:
Okay, sounds good. Let's keep it as is.